### PR TITLE
chore: clarify why metrics are disabled

### LINKED ISSUE #129: Disable metrics by default
By default, we want to only collect audit logs.

Go through all modules and make sure metrics are disabled by default. Will be adding a new variable to enable metrics if needed.

this is in reference to another [issue](https://github.com/equinor/terraform-baseline/issues/44)

### PR DESCRIPTION
**Pull request by kov117**

### CODE REVIEW

**Review by hknutsen — COMMENTED** [review]
The diagnostic setting you're changing is for the automatically created `master` database.

This database does not actually have real support for diagnostic settings, however the diagnostic setting is used to route audit logs configured by the `azurerm_mssql_server_extended_auditing_policy` resource to a Log Analytics workspace.

So this diagnostic setting is actually a special case where we don't want to allow users to set metric categories, since they're not actually supported.

Thus, I think that this PR can be closed.

**Comment by hknutsen** [comment]
... or, even better, remove your code changes but add a comment in the code clarifying why metrics are not configured 😉

**Comment by kov117** [comment]
@hknutsen something like this ?

File: main.tf [hunk 1]
```diff
@@ -75,6 +75,8 @@ resource "azurerm_monitor_diagnostic_setting" "this" {
     }
   }
 
+# The metrics will not be configured, because we don't want to allows users to set metric categories.
+# Since the metrics are not actually supported by this database, but we use it to route audit logs.
```
Comment (hknutsen) [inline main.tf]: Clarify:

```suggestion
# Metrics are not supported at the master database scope.
```

**Review by hknutsen — COMMENTED** [review]
See suggestion 🦖

**Comment by hknutsen** [comment]
Purpose of PR has changed. Updating title.

**Review by hknutsen — APPROVED** [review]

**Pull request merged**